### ISSUE TITLE
Please provide an RSS Feed

### ISSUE THREAD

**Issue by splitbrain** [uxai/non-profit-bloggers#10]
It would be great if you could provide a RSS feed of blogs added to your list, so people can subscribe to it.

**Comment by uxai** [comment]
Good idea :) Thanks for the suggestion. I have added a column for the table. I will go back later and add peoples RSS feeds when I have time.

**Comment by splitbrain** [comment]
Oh, I think you misunderstood. I want an RSS feed of the list **you** are aggregating here, so I don't have to manually check the repo every few days to discover new blogs.

**Comment by uxai** [comment]
Ah, I understand. I will reopen this to remind me to do something of this sort. I am not sure exactly how to do this. But I will definitely look into it!

**Comment by vincode-io** [comment]
@uxai probably the easiest way for you to get an RSS feed up and running is to use [Github Pages with Jekyll](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll). Then you just do a blog entry every time you add a new feed to your README.md.

If it were me, I'd just point the README.md to the Non-Profit Bloggers blog you created. That way you wouldn't have to manage the list in two separate places. [Blogroll](blogroll.org) works this way.

It looks like you have some scripting skills, so you might even be able to handle #14 at the same time by writing a custom Jekyll plugin to generate the OPML file. You would need to define a custom [Front Matter](https://jekyllrb.com/docs/front-matter/) variable. [Here is an example of how to do this.](https://chenhuijing.com/blog/handling-articles-on-external-sites).

Then you would write the plugin. This [OPML Feed Generator for Jekyll](https://github.com/mparuszewski/jekyll-opml-generator) project does something similar, but is way overkill for what you would need. Just chop the pieces out you don't need and do some modifications to get the Front Matter info you need and it would be easy to maintain your list going forward.
